src/ui/qgsattributesforminitcode.ui

```diff
@@ -7,95 +7,96 @@
     <x>0</x>
     <y>0</y>
     <width>400</width>
-    <height>300</height>
+    <height>283</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Python Init Code Configuration</string>
   </property>
-  <layout class="QGridLayout" name="gridLayout">
-   <item row="1" column="0">
-    <widget class="QWidget" name="mInitFunctionContainer" native="true">
-     <property name="sizePolicy">
-      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
-       <horstretch>0</horstretch>
-       <verstretch>0</verstretch>
-      </sizepolicy>
+  <layout class="QGridLayout" name="gridLayout_2">
+   <item row="0" column="0">
+    <widget class="QLabel" name="label_3">
+     <property name="toolTip">
+      <string>The function code of the function can be loaded from the source code entered 
+in this dialog,  from an external python file or from the environment (for example 
+from a plugin or from startup.py).
+
+An example is:
+
+          rom qgis.PyQt.QtWidgets import QWidget
+
+          def my_form_open(dialog, layer, feature):
```

Comment:
ħow about removing the "my" prefix from the code snippet? From what i've understood we are avoiding it in QGIS code so better give that example to others :)